EnglishTranslation-master/ITEM.tsv

```diff
@@ -10124,7 +10124,7 @@ ITEM_20160224_010123	Contains silver. This item will be removed after the server
 ITEM_20160224_010124	Intermediate Silver Chest
 ITEM_20160224_010125	Advanced Silver Chest
 ITEM_20160224_010126	Settlement Support Potion
-ITEM_20160224_010127	Increases your movement speed by 2, physical and magic attack by 50 for 30 minutes. {nl}Right-click to use.{nl}This item will be removed after the server maintenance of March 4th.
+ITEM_20160224_010127	Increases your movement speed by 2. Physical and magic attack is increased by 50 for 30 minutes.{nl}Right-click to use.{nl}This item will be removed after the server maintenance of March 4th.
```

Comment:
This can be phrased better. Now one might think that this increases movement speed by 2. (Period) And physical/magic attack by 50 for 30 minutes. 

What about: 

Increases your movement speed and your physical/magic attack by 2 and 50, respectively. 

?